.github/workflows/docker_utils.yml

```diff
@@ -75,9 +79,9 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
-            type=ref,event=branch,suffix=${{ inputs.features == 'tce-direct,sequencer' && '-direct' || ''}}
-            type=ref,event=tag,suffix=${{ inputs.features == 'tce-direct,sequencer' && '-direct' || ''}}
-            type=ref,event=pr,suffix=${{ inputs.features == 'tce-direct,sequencer' && '-direct' || ''}}
+            type=ref,event=branch,suffix=${{ inputs.docker_tag != '' && format('-{0}', input.docker_tag) || '' }}
```

Comment:
Instead of add yet another input, why don't we just set the suffix as the value of `features`? Then, it's clear what is behind an image and we prevent us from ending up with broken images (e.g., an image with `checker` docker tag but without the `checker` feature because of human error).

Reply:
If we want to use the `features` as a suffix, we need to parse the input and replace any `,` with `_` just for the tag.
Also, our base images will be:

- `topos-network/topos:tce-direct_sequencer`
- `topos-network/topos:tce_sequencer`
- `topos-network/topos:checker`

The more feature we will add the more long the tag will be, it also prevents nested feature activation but we can manage that. I'm ok to go with your approach but we need to think about what the end user will use.

Questions:
- How do we manage the root image if the suffix is always the `features` ?
- How do we manage feature changes over time (like rename, etc) ?

Reply:
> How do we manage the root image if the suffix is always the features ?
We can change the ternary condition to add the `${features}` suffix only if `features !== DEFAULT_FEATURES`.

> How do we manage feature changes over time (like rename, etc) ?
I guess a change in the name of a feature is as big on the native binary than it is on docker images, so I don't see a problem with it in the docker context.